client/views/connection_list.tpl

```diff
@@ -0,0 +1,11 @@
+<p class="clearfix">
+	{{#if id includeZero=true}}
+		<button class="btn pull-right" id="connection_{{id}}" value="{{socketId}}">Disconnect</button>
+	{{/if}}
+
+	<strong>{{hostname}}</strong>
+	<br>
+	<a class="" href="https://geoiptool.com/?ip={{ip}}">{{ip}}</a>
```

Comment:
Not sure if we should rely on a third party service -- do we need to link to the tool?